mod_reforged/hooks/entity/tactical/player.nut

```diff
@@ -24,12 +24,9 @@
 		{
 			if (attribute == ::Const.Attributes.COUNT) continue;
 
-			local attributeMin = ::Const.AttributesLevelUp[attribute].Min;
-			if (this.m.Talents[attribute] >= 1) attributeMin++;
-			if (this.m.Talents[attribute] == 3) attributeMin++;
-
+			local attributeMin = ::Const.AttributesLevelUp[attribute].Min + ::Math.min(this.m.Talents[attribute], 2);
 			local attributeMax = ::Const.AttributesLevelUp[attribute].Max;
-			if (this.m.Talents[attribute] >= 2) attributeMax++;
+			if (this.m.Talents[attribute] == 2) attributeMax++;
```

Comment:
What about 3 Stars? They can roll up to +1 maximum attribute. You seem to have removed that part now

Reply:
Ah my bad, that's supposed to be `== 3`. I'll fix.